rewrite-java-11/src/main/java/org/openrewrite/java/Java11ParserVisitor.java

```diff
@@ -1537,26 +1536,38 @@ private Space statementDelim(@Nullable Tree t) {
             };
 
             JavaType.Method.Signature genericSignature;
-            if (genericSymbol.type instanceof com.sun.tools.javac.code.Type.ForAll) {
-                genericSignature = signature.apply(((com.sun.tools.javac.code.Type.ForAll) genericSymbol.type).qtype);
+            if (methodSymbol.type instanceof com.sun.tools.javac.code.Type.ForAll) {
+                genericSignature = signature.apply(((com.sun.tools.javac.code.Type.ForAll) methodSymbol.type).qtype);
             } else {
-                genericSignature = signature.apply(genericSymbol.type);
+                genericSignature = signature.apply(methodSymbol.type);
             }
 
             List<String> paramNames = new ArrayList<>();
-            for (Symbol.VarSymbol p : genericSymbol.params()) {
+            for (Symbol.VarSymbol p : methodSymbol.params()) {
                 String s = p.name.toString();
                 paramNames.add(s);
             }
 
+            List<JavaType.FullyQualified> exceptionTypes = new ArrayList<>();
```

Comment:
Curious if these should be JavaType.FullyQualified or JavaType.Class?

Reply:
Also wasn't sure what the behavior should be when the exception class cannot be resolved. Right now, it just skips any exceptions that could not be resolved.

Reply:
Re: FullyQualified vs Class - I'm not completely sure but I suspect `Class` is fine.

RE: unresolvable classes. If you know their fully qualified name I might vote to construct a synthetic type for them. If that isn't feasible then skipping is probably the thing to do

Reply:
Thanks for the feedback. I changed Java11ParserVisitor to create JavaType.Class for exceptions. Non-resolved instances will then have a synthetic created via JavaType.Class.build()